go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ethereum/go-ethereum
 
-go 1.23.0
+go 1.24.0
+
+toolchain go1.24.3
```

Comment:
Is it possible to omit the `toolchain` here? Upstream doesn't specify it and if possible we also try to avoid it. IIRC it makes it easier to import and use it in the monorepo.

```suggestion
```